data/service/authorization-manager.js

```diff
@@ -259,38 +282,31 @@ var AuthorizationManager = Montage.specialize(/** @lends AuthorizationManager.pr
      * @method
      * @argument {DataService} dataService - A dataService for which to get an authorization
      */
-
-
     authorizeService: {
         value: function (dataService, didFailAuthorization) {
             var self = this,
                 authorizationPromises = [];
 
-
             if (dataService.authorizationPolicy === AuthorizationPolicy.NONE) {
                 return Promise.resolve(null);
             } else {
-
-                //[TJ] This will only work for data services with a single authorization-service
+                //[TJ] This will only work for data services with a single authorization-service    
```

Comment:
@marchant 
The comment has a kernel of truth, but isn't wholly accurate. I removed the comment in code to avoid further confusion (https://github.com/montagejs/montage/commit/063f5465fdb34c8ef259f842559db008810de0b2) and will clarify below....

`AuthorizationManager` can handle a `DataService` with multiple authorization services, but only the case where the `DataService` requires *all* of the authorization services to resolve before it considers itself authorized. It does not support the user choosing only one of the provided authorization services. This is a case we must handle. Further, I think that case was the original intention. 

That said, this PR contains useful additions that I do not want to delay for the sake of the multiple auth-service issue. I would prefer to move that to its own issue/PR.